spec/requests/cloud_volumes_spec.rb

```diff
@@ -141,22 +143,25 @@
     end
 
     it 'safe_delete will raise an error if the cloud volume does not support safe_delete' do
-      ems    = FactoryBot.create(:ems_autosde, :name => "Autosde")
-      volume = FactoryBot.create(:cloud_volume, :ext_management_system => ems, :name => "my_volume")
+      ems    = FactoryBot.create(:ext_management_system)
+      volume = FactoryBot.create(:cloud_volume, :ext_management_system => ems)
+      stub_supports_not(volume, :safe_delete)
+
       api_basic_authorize(action_identifier(:cloud_volumes, :safe_delete, :resource_actions, :post))
 
       post(api_cloud_volume_url(nil, volume), :params => {"action" => "safe_delete"})
       expect_bad_request(/Safe Delete for Cloud Volume.*not available/)
     end
 
     it "can safe delete a cloud volume as a resource action" do
-      ems = FactoryBot.create(:ems_autosde, :name => "Autosde")
-      volume1 = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => ems, :name => "my_volume")
```

Comment:
Also not sure why this was volume1 when there was no volume2, switched it back to just `volume` like the rest of these tests